network_diffusion/seeding/drivernode_selector.py

```diff
@@ -0,0 +1,82 @@
+"""A definition of the seed selector based on driver nodes."""
+
+from typing import Any, List, Optional
+
+import networkx as nx
+
+from network_diffusion.mln.actor import MLNetworkActor
+from network_diffusion.mln.mlnetwork import MultilayerNetwork
+from network_diffusion.seeding.base_selector import BaseSeedSelector
+from network_diffusion.tpn.functions import compute_driver_nodes
+from network_diffusion.tpn.tpnetwork import TemporalNetwork
+from network_diffusion.utils import BOLD_UNDERLINE, THIN_UNDERLINE
+
+
+class DriverNodeSelector(BaseSeedSelector):
+    """Driver Node based seed selector."""
+
+    def __init__(self, method: Optional[BaseSeedSelector]) -> None:
```

Comment:
@uniooo - I modified your class so that it's now more flexible. Instead of relying on `method` passed as a string which is being matched with proper sub-selector in the next step, I'm passing immediately an object of sub-selector. With this approach we're not vulnerable to typos in the string, and we are not obliged to update a list of matching selectors each time a new one will be added to our library.

Reply:
That's great!